src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Sorts all persons in the address book to the user.
```

Comment:
Good documentation before the class implementation to have a overview of what the class will be doing.